### PR TITLE
Updated NextVersion.md to include AppUI ViewSelector enhancements

### PR DESCRIPTION
**Pull request by kconner1**

### CODE REVIEW

File: docs/changehistory/NextVersion.md [hunk 1]
```diff
@@ -80,6 +81,10 @@ Below is a list of the changes from this move, some of these new access point ma
  UiFramework.childWindowManager.closeChildWindow | UiFramework.childWindows.close
  UiFramework.backstageManager | UiFramework.backstage
 
+### ViewSelector Enhancements
+
+Search functionality has been added to the `ViewSelector` component, allowing for additional ease when navigating between iModel views. Search functionality is optional and enabled by default. To disable search functionality, set the `searchBox` property of `ViewSelector` to false.
```
Comment (pmconne) [inline docs/changehistory/NextVersion.md:86]:
```suggestion
Search functionality has been added to the [ViewSelector]($appui-react) component, allowing for additional ease when navigating between iModel views. Search functionality is optional and enabled by default. To disable search functionality, set the [ViewSelector.searchBox]($appui-react) to `false`.
```

Comment (pmconne) [reply docs/changehistory/NextVersion.md:86]: Consider using doc links so people can quickly jump to more information on the doc site.

**Review by pmconne — APPROVED** [review]

File: docs/changehistory/NextVersion.md (hunk 1 above)
Comment (pmconne) [reply docs/changehistory/NextVersion.md]: I guess by "searchBox property of ViewSelector" you must have meant ViewSelectorProps, right?

File: docs/changehistory/NextVersion.md [hunk 2]
```diff
@@ -80,6 +81,10 @@ Below is a list of the changes from this move, some of these new access point ma
  UiFramework.childWindowManager.closeChildWindow | UiFramework.childWindows.close
  UiFramework.backstageManager | UiFramework.backstage
 
+### ViewSelector Enhancements
+
+Search functionality has been added to the [ViewSelector]($appui-react) component, allowing for additional ease when navigating between iModel views. Search functionality is optional and enabled by default. To disable search functionality, set the [ViewSelector.searchBox]($appui-react) to `false`.
```
Comment (pmconne) [inline docs/changehistory/NextVersion.md:86]:
```suggestion
Search functionality has been added to the [ViewSelector]($appui-react) component, allowing for additional ease when navigating between iModel views. Search functionality is optional and enabled by default. To disable search functionality, set the [ViewSelectorProps.searchBox]($appui-react) to `false`.
```

File: docs/changehistory/NextVersion.md (hunk 1 above)
Comment (kconner1) [reply docs/changehistory/NextVersion.md]: Yes, that is correct

**Pull request merged**